srtcore/core.h

```diff
@@ -283,6 +294,14 @@ class CUDT
     // a different channel.
     void skipIncoming(int32_t seq);
 
+
+    SRTU_PROPERTY_RO(bool, closing, m_bClosing);
+    SRTU_PROPERTY_RO(CRcvBuffer*, rcvBuffer, m_pRcvBuffer);
+    SRTU_PROPERTY_RO(bool, isTLPktDrop, m_bTLPktDrop);
+    SRTU_PROPERTY_RO(bool, isSynReceiving, m_bSynRecving);
```

Comment:
To preserve similar naming (see isTLPktDrop, isSynReceiving), better to have `isClosing`